README.md

```diff
@@ -180,7 +180,8 @@ The Azure MCP Server supercharges your agents with Azure context. Here are some
 * List Azure Foundry models
 * Deploy foundry models
 * List foundry model deployments
- 
+* List knowledge indexes
```

Comment:
nit: indexes or indices?

Reply:
I noticed that `indexes` is also used in AI Search.

I will retain it in this PR, and we can update all instances later.